lang/fr-on.js

```diff
@@ -0,0 +1,110 @@
+/* eslint quotes: 0 */
+
+export default {
+	"components.alert.close": "Fermer l'alerte",
+	"components.breadcrumbs.breadcrumb": "Chemin de navigation",
+	"components.calendar.notSelected": "Non sélectionné(e)",
+	"components.calendar.selected": "Sélectionné(e).",
+	"components.calendar.show": "Afficher {month}",
+	"components.count-badge.plus" : "{number}+",
+	"components.dialog.close": "Fermer cette boîte de dialogue",
+	"components.dropdown.close": "Fermer",
+	"components.filter.clear": "Effacer",
+	"components.filter.clearAll": "Effacer tout",
+	"components.filter.clearAllAnnounce": "Effacement de tous les filtres en cours",
+	"components.filter.clearAllDescription": "Effacer tous les filtres",
+	"components.filter.clearAnnounce": "Effacement des filtres pour : {filterName} en cours",
+	"components.filter.clearAnnounceSingle": "Effacement des filtres en cours",
+	"components.filter.clearDescription": "Effacer les filtres pour : {filterName}",
+	"components.filter.clearDescriptionSingle": "Effacer les filtres",
+	"components.filter.loading": "Chargement des filtres",
+	"components.filter.filterCountDescription": "{number, plural, =0 {Aucun filtre appliqué.} one {1 filtre appliqué.} other {{number} filtres appliqués.}}",
+	"components.filter.filters": "Filtres",
+	"components.filter.noFilters": "Aucun filtre disponible",
+	"components.filter.searchResults": "{number, plural, = 0 {Aucun résultat de recherche} one {1 résultat de recherche} other {{number} résultats de recherche}}",
```

Comment:
Weirdly this is identical to the `fr` one except they put a space between `=` and `0` in `=0`. Will remove this anyway.